src/cirrus/plugins/management/deployment.py

```diff
@@ -265,6 +270,61 @@ def get_execution_by_payload_id(self, payload_id):
 
         return self.get_execution(exec_arn)
 
+    def run_workflow(
+        self,
+        payload: dict,
+        timeout: int = 3600,
+        force_rerun: bool = False,
+        out_path: str = "",
+    ) -> dict:
+        """
+
+        Args:
+            deployment (Deployment): where the workflow will be run.
+
+            payload_path (str): path to payload to pass to the deployment to kick off the workflow.
+
+            timeout (Optional[int]): - upper bound on the number of seconds to poll the deployment before
+                                       considering the test failed.
+
+            out_path (Optional[str]): - path to write the output or error message to.
+
+        Returns:
+            dict containing output payload or error message
+
+        """
+        payload = ProcessPayload(payload)
+        if force_rerun:
+            payload["id"] = f"{payload['id']}-{time_ns()}"
```

Comment:
Why change the ID rather than set the replace flag in the payload?

Reply:
Interesting.  So, that's a legacy artifact of the deployment I'm working with, which didn't utilize `ProcessPayload`s until last year.  Seems like this gets backed out.

Reply:
The replace flag has been supported for as long as we've had a state database, I believe, which is independent of the introduction of the `ProcessPayload` class (which itself existed in a prior form `Catalog`, but that was misleadingly named, hence the change to `ProcessPayload).

Anyway, my feeling here is that the user should be able to set the replace flag on the payload via this command:

```suggestion
            payload.process["replace"] = True
```

And if you need to have a unique ID you should use a payload templating step prior to this command.